### PR TITLE
Added whitepaper describing fingerprinting Math

### PR DESCRIPTION
**Pull request by yfarjoun**
CheckFingerprints, CrosscheckFingerprints, and IdentifyContaminant are based on some probabilistic calculations that are not properly represented anywhere, really. 

This PR adds a short paper that describes the math.

### CODE REVIEW

**Comment by yfarjoun** [comment]
@takutosato and @madduran are you going to get to this or should I find someone else to review?

File: docs/fingerprinting/main.tex [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+For the calculation we will need the prior on the haplotypes, $p(\theta)$ (which can be calculated from the haplotype frequency by assuming Hardy-Weinberg equilibrium.
+
+We assume that the underlying samples have (unknown) haplotypes $\theta$ and $\phi$, mixed with proportion $1-c$ and $c$, respectively. The data collected from this mixture, is $x$.
+Then we can write: 
+\begin{equation}
+p(x|c,\theta)=\sum_\phi p(x|c,\theta,\phi)p(\phi)
+\end{equation}
+
+
+...
```
[the remark this answers is not in the document]
Comment (yfarjoun) [reply docs/fingerprinting/main.tex]: (to remind me to finish my thought...thanks!)

**Comment by yfarjoun** [comment]
a bit slow...but I responded...please re-read.

File: docs/fingerprinting/main.tex [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+\end{align*}
+Substituting these expressions into \eqref{odds} and re-distributing yields
+\begin{equation*}
+\boxed{\prod_i \left(\frac{\sum_{\theta_i} p(x_i \, | \, \theta_i) \, p(\theta_i) \, \sum_{\varphi_i} p(y_i \, | \, \varphi_i) \, p(\varphi_i)}{\sum_{\theta_i = \varphi_i} p(x_i \, | \, \theta_i) \, p(y_i \, | \, \varphi_i) \, p(\theta_i)}\right) \cdot \frac{p(s=1)}{p(s=0)}.}
+\end{equation*}
+
+\section{Haplotype Likelihoods}
+In this section we describe how the haplotype likelihood $p(x|\theta)$ can be computed for various kinds of data.
+\subsection{Sequence data}
+For sequence data, we assume that the data come from a single individual (i.e. not contaminated, see subsection below) and that there is no reference bias (can correct for that, but it's rarely needed)
```
[the remark this answers is not in the document]
Comment (madduran) [reply docs/fingerprinting/main.tex]: still missing a period here

**Comment by takutosato** [comment]
👍

File: docs/fingerprinting/main.tex (hunk 2 above)
[the remark this answers is not in the document]
Comment (yfarjoun) [reply docs/fingerprinting/main.tex]: data is plural

**Pull request merged**